Story/StoryLobbyData.cs

```diff
@@ -98,6 +98,11 @@ internal override void ReadTo(ResourceData data)
                     storySession.saveState.deathPersistentSaveData.karma = karma;
                     storySession.saveState.deathPersistentSaveData.reinforcedKarma = reinforcedKarma;
                     storySession.saveState.theGlow = theGlow;
+                    if ((RWCustom.Custom.rainWorld.processManager.currentMainLoop is RainWorldGame rainWorldGame))
+                    {
+                        if(rainWorldGame.Players[0].realizedCreature != null)
+                            (rainWorldGame.Players[0].realizedCreature as Player).glowing = theGlow;
+                    }
```

Comment:
I see, so the RPC updates the lobby owner who owns storydata and the rest of the players gets it through here, this is perfect.

I think players might not see others glowing though? Are remote players's `glowing` set anywhere? I don't think it's synched by realizedplayerstate.

Reply:
added